policies/trademark_registration.md

```diff
@@ -0,0 +1,28 @@
+## Trademark Registration for Projects
+
+While projects usually retain a common-law trademark to their marks, it can be benefitial
+to do an actual trademark registration.
+
+Since such a registration incurs cost both to obtain the trademark, and maintain it over
+time, HPSF is not doing it for each member project automatically. Instead projects
+have to request trademark registration from the governing board.
+
+#### Criteria for Consideration
+
+- “Established” or higher project lifecycle
+  - In exceptional cases “Emerging” projects can be considered
+- Demonstrate need/benefit of doing full trademark
+  - Already ran into issues with invalid use of trademark outside of the own project e.g.:
+    - Research projects name things invalid
+    - Commercial products named invalid
+    - Customers/users confused about what is official (i.e. LF) vs. third party effort
+  - Strong likelihood of issues coming up e.g.:
+    - Large diverse user or developer community with many institutions
+    - Derivative projects are very common
+
+#### Process
+
+- Request to Governing Board by project leads
+- (Confidential) presentation laying out how the criteria above apply to the project
+- Governing Board reviews and votes on the request
```

Comment:
```suggestion
1. the project leads request the Governing Board to consider the matter,
  - this can be done through the TAC representatives at the board
2. (Confidential) the project lead present their needy to the board, laying out ow the criteria above apply
  - the board will schedule this presentation at the earliest possible board meeting, and no more than two month after the project request
3. the Governing Board reviews and votes on the request
4. if the request is accepted, the foundation will start the process to legally register the trademark in the region of interest as determined by the board
```